### PR TITLE
chore(studio): re-implemented export to archive util function

### PR DESCRIPTION
**Pull request by laurentlp**
This PR simply re-implements the export to archive buffer ghost utility function to the `Instance` util.

To test this feature, you can upload your bot to the cloud and see if it works as expected.

### CODE REVIEW

**Review by Raid55 — APPROVED** [review]
LGTM, only a conflict with yarn.lock then good to go

File: packages/studio/server/src/studio/nlu/nlu-service.ts [hunk 1]
```diff
@@ -148,29 +149,26 @@ export class NLUService {
...
     const modelsFolder = 'models'
-    await this.ghost.forBot(botId).deleteFolder(modelsFolder)
+    await Instance.deleteDir(modelsFolder)
 
     for (const lang of Object.keys(botConfig.nluModels)) {
       const modelId = botConfig.nluModels[lang]
       const modelWeights = await bot.downloadModelWeights(botId, modelId)
 
-      await this.ghost.forBot(botId).upsertFile(modelsFolder, `${modelId}.model`, modelWeights)
+      await Instance.upsertFile(path.join(modelsFolder, `${modelId}.model`), modelWeights)
```
Comment (EFF) [inline packages/studio/server/src/studio/nlu/nlu-service.ts:170]: **question**: have you verified modelWeights encoding ? i seems you save it as raw bytes (or downloadModelWeights already pareses it). I imagine you tried to upload it on cloud and works.

File: packages/studio/server/src/studio/utils/bpfs.ts [hunk 2]
```diff
@@ -93,5 +96,38 @@ export const Instance: bpfs = {
   },
   moveFile(fromPath: string, toPath: string): Promise<void> {
     return fse.move(resolvePath(fromPath), resolvePath(toPath))
+  },
+  async exportToArchiveBuffer(): Promise<Buffer> {
```
Comment (EFF) [inline packages/studio/server/src/studio/utils/bpfs.ts:100]: **nitpick**: although writing on disk is totally fine on the studio app we could have done this in all in mem. (not to be fixed)

**Review by EFF — APPROVED** [review]
LGTM

File: packages/studio/server/src/studio/nlu/nlu-service.ts (hunk 1 above)
Comment (laurentlp) [reply packages/studio/server/src/studio/nlu/nlu-service.ts:170]: I haven't checked the encoding, but I have tried to upload it and it worked

**Pull request merged**